internal/devserver/ts_devserver.bzl

```diff
@@ -164,7 +164,8 @@ ts_devserver = rule(
         ),
         "_requirejs_script": attr.label(allow_single_file = True, default = Label("@build_bazel_rules_typescript_devserver_deps//node_modules/requirejs:require.js")),
         "_devserver": attr.label(
-            default = Label("//devserver"),
+            # In development use Lavel("//devserver")
```

Comment:
This looks like a typo. `Label`?

Also even though this PR builds Windows binaries, Windows devserver doesn't work 😄we need to move #327 forward to get it work. Nevertheless, makes sense to build the binaries already. Very cool!